apps/web/src/app/(dashboard)/nav-bar.tsx

```diff
@@ -0,0 +1,40 @@
+'use client'
+
+import React, { useState } from "react";
+import NavBar from "@/components/shared/nav-bar/nav-bar";
+import { Links } from '@/utils/constants'
+import HomeIcon from '@mui/icons-material/Home';
+import BookOpenTextIcon from '@mui/icons-material/ImportContacts';
+import FileUploadIcon from '@mui/icons-material/FileUpload';
+import { Theme } from "@mui/material"
+
+export type NavbarItemProps = {
+  name: string
+  link: string
+  icon: React.FC
+}
+
+export const navbarItems = [
+  { name: 'Dashboard', link: Links.HOME, icon: HomeIcon },
+  { name: 'My Learning', link: Links.COURSES, icon: BookOpenTextIcon },
+  { name: 'Upload', link: Links.UPLOAD, icon: FileUploadIcon },
+] as NavbarItemProps[]
+
+const DashboardNavBar = () => {
+  const [mobileOpen] = useState(false);
+  const handleDrawerToggle = () => {
+  };
+
+  return (
+    <NavBar
+      zIndex={(theme: Theme) => ({
+        xs: mobileOpen ? theme.zIndex.drawer - 1 : theme.zIndex.drawer + 1,
+        sm: theme.zIndex.drawer + 1
+      })}
+      onMenuIconClick={handleDrawerToggle}
```

Comment:
you can remove the empty `handleDrawerToggle`

```suggestion
      onMenuIconClick={() => {}}
```